public/app/core/components/TransformersUI/MergeTransformerEditor.tsx

```diff
@@ -7,6 +7,10 @@ export const MergeTransformerEditor: React.FC<TransformerUIProps<MergeTransforme
   options,
   onChange,
 }) => {
+  if (input.length <= 1) {
+    // Show warning that merge is useless only apply on a single frame
+    return <div>Merge has no effect when applied on a single frame.</div>;
```

Comment:
Maybe this could you the Alert component instead?

Reply:
I's basically an adjusted c&p from https://github.com/grafana/grafana/blob/cc4d301d50440c529ce96da8afe999bb46991403/public/app/core/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx#L76